adb_client/src/models/remount_info.rs

```diff
@@ -0,0 +1,39 @@
+use crate::Result;
+
+#[derive(Debug)]
+/// Information about remount operation
+pub struct RemountInfo {
+    /// Path that was remounted
+    pub path: String,
+    /// Mode that was used for remounting
+    pub mode: String,
+}
+
+impl RemountInfo {
+    pub(crate) fn from_str(s: &str) -> Result<Self> {
+        let parts: Vec<&str> = s.split_whitespace().collect();
+        if parts.len() == 4 && parts[0] == "Using" && parts[2] == "for" {
```

Comment:
Is it here?
I'm not sure about how to implement this in a more rusty way.
Any suggestion?